src/OrbitGl/TimeGraph.cpp

```diff
@@ -163,15 +163,19 @@ void TimeGraph::VerticalZoom(float zoom_value, float mouse_relative_position) {
   const float capped_ratio = old_scale / layout_.GetScale();
 
   const float world_height = viewport_->GetVisibleWorldHeight();
-  const float y_mouse_position =
-      viewport_->GetScreenTopLeftInWorld()[1] - mouse_relative_position * world_height;
-  const float top_distance = viewport_->GetScreenTopLeftInWorld()[1] - y_mouse_position;
+  const float mouse_position_in_world_y =
+      viewport_->GetScreenTopLeftInWorld()[1] + mouse_relative_position * world_height;
+  const float mouse_position_in_screen =
```

Comment:
Nit: Seems `mouse_position_in_screen_y` would be better here as we are computing the y position.

Reply:
Yes, of course! Thanks for catching it!

Reply:
(Although it was here before) The second param of `VerticalZoom` is named as `mouse_ratio` in the header declaration but is `mouse_relative_position` here.  Which one is correct? 

Could you help to make them consistent since you are touching this place?

Reply:
What you call "screen" here isn't really screen, if so it should go through the `viewport_->ScreenToWorldPos` method - *but* you're subtracting it from `new_mouse_position_in_world_y` below, which could be wrong if world and screen were scaled differently...

This is what I'd call "relative_mouse_position_in_world_y"... as in "relative to the top left"...